src/botPage/bot/TradeEngine/Proposal.js

```diff
@@ -88,7 +88,10 @@ export default Engine =>
                         throw error;
                     })
                 )
-            ).catch(e => this.$scope.observer.emit('Error', e));
+            ).catch(({ error = {} }) => {
+                this.$scope.observer.emit('Error', { name: error.code, ...error });
```

Comment:
@sandeep-deriv usually we also handle it with '`throw new TrackJSError(code, message, error);`' in the **.catch()** 
Can we add it also like this? Check please another place too in [src/botPage/bot/TradeEngine/index.js](https://github.com/deriv-com/binary-bot/pull/244/files#diff-304089dea932d11dd081f0be1a733c8de4d45964da86b2d9ced09d24ab5873a9)
```suggestion
                this.$scope.observer.emit('Error', { name: error.code, ...error });
                throw new TrackJSError(error.code, error.message, error);
```

Reply:
Not all errors should be sent to TrackJS. Some errors like related to minimum stake input shouldn't be included.

Reply:
@maryia-matskevich-deriv we don't need to add it to trackjs ignore list as we are handing the error here.
Thanks for your concerns 🍅